### PR TITLE
DEL-203:Moving OCM tool to Delorean repo

### PR DESCRIPTION
**Pull request by STEPHINRACHEL**
**Motivation**
https://issues.redhat.com/browse/DEL-203

**What**
Moved OCM tool which is used to create an  OSD cluster and trigger rhmi installation using addon flow from rhmi-utils repo to delorean repo.

**Why**
To keep delorean repo as a single place to store all the Delorean tools

**To Verify**
1. Check out this branch 
2. Follow the instruction in the  README file under docs/ocm/

### CODE REVIEW

**Comment by StevenTobin** [comment]
:eyes:

**Comment by StevenTobin** [comment]
So it brings up a cluster successfully and kicks off the addon installation successfully. But the ingress operator seems to be in a bad state, I highly doubt it's due to anything here but I just want to try it manually to make sure first.

**Comment by STEPHINRACHEL** [comment]
> So it brings up a cluster successfully and kicks off the addon installation successfully. But the ingress operator seems to be in a bad state, I highly doubt it's due to anything here but I just want to try it manually to make sure first.

**Comment by STEPHINRACHEL** [comment]
closed PR by mistake

**Comment by StevenTobin** [comment]
Ok so following the instructions in the README seems to result in a broken cluster. ~11 alerts firing and the authentication and ingress operators reporting degraded state.

When I bring up a cluster manually using ocm I don't see these issues, did you try the commands out successfully?

**Comment by STEPHINRACHEL** [comment]
@StevenTobin I just tried out only the cluster creation as I did not make any code changes. I will try with the steps under https://github.com/integr8ly/rhmi-utils in order to see if the issue you mentioned happens there too.

File: openshift-ci/OWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- davidffrench
+- pawelpaszki
+- psturc
+
+reviewers:
+- mikenairn
+- jessesarn
+- davidffrench
+- pawelpaszki
+- psturc
```
Comment (mikenairn) [inline openshift-ci/OWNERS:15]: You can remove this OWNERS file

**Comment by StevenTobin** [comment]
Cluster came up fine using the tooling
before

But after using make ocm/install/rhmi-addon the cluster became inaccessible and a number of alerts are triggering
alerts

**Comment by mikenairn** [comment]
I've never tried the addon part of these scripts before tbh, @psturc Is this an issue you have seen before?

**Comment by psturc** [comment]
We're using the tooling in rhmi-utils repo in nightly pipelines and I haven't seen this issue before... Also have no idea what could cause this.

`make ocm/install/rhmi-addon` just enables the rhmi addon and creates mandatory resources in `redhat-rhmi-operator` namespace to trigger the installation

I'll try to verify these changes locally.

**Comment by psturc** [comment]
I've hit the issue, but it is apparently not related to the `make ocm/install/rhmi-addon` command.
I just provisioned a cluster and waited couple of minutes and the alerts started firing.

Screenshot 2020-04-16 at 16 21 50

When I tried a second time, I used a different region (eu-central1) and custom cluster name (psturc-test) and the cluster still seems to be fine after couple of hours, even after running `make ocm/install/rhmi-addon`.

🤷‍♂️

**Pull request merged**